README.md

```diff
@@ -19,60 +19,43 @@ yarn install
 
 Client for StreamDB uses generated gRPC stubs to provide fast access to data. 
 
-#### Documentation 
-Link to documentation can be found here.
-[https://buf.build/proximaone/streamapis/docs/main/messages.v1alpha1](https://buf.build/proximaone/streamapis/docs/main/messages.v1alpha1)
-
-
-### Sample 
-
-Check out full solution at https://github.com/proxima-one/proxima-samples/tree/master/eth-blockheaders-stream
+### Installation 
 
 ```typescript 
-import { StreamClient } from "@proximaone/stream-client-js";
-import { map } from "rxjs";
-
-async function main() {
-  const client = new StreamClient("streamdb.cluster.prod.proxima.one:443");
-
-  const blockHeadersStream = "eth-main-headers";
-
-  // fetch first 1000 block header events
-  const firstBlockHeaders = await client.getNextMessages(
-    blockHeadersStream,
-    {messageCount: 1000},
-  );
 
-  console.log(`Fetched first ${firstBlockHeaders.messagesList.length} messages`);
-
-  // continue reactively consuming stream messages starting from last fetched message
+import { StreamClient } from "../src";
```

Comment:
I think we can even remove the code structure like imports now when we have separate code samples. Leaving only something like this:
```typescript
const client = new StreamClient({});
const name = "eth-main-blockheader";
const stream = await client.getStream(name);
console.log(`Success! Fetched stream ${stream.toString()}`);
const height = stream.stats.start.height;
const offset = await client.findOffset(name, {height: Number(height)});

const pausable = await client.streamEvents(name, offset);
pausable.observable.pipe(map(event => {
  console.log(event.undo ? "undo" : "", event.payload);
}));
```

By the way, not the simplest code to just read the stream from the beginning. Maybe we should add some fancy helpers like `readStreamFromStart` in the future.